docs/README.md

```diff
@@ -157,12 +159,24 @@ Check out [Kubernetes documentation](https://kubernetes.io/docs/tasks/inject-dat
 
 ### Configuring Lookback
 
-The lookback strategy determines how the agent handles existing files on startup. This strategy is determined by `LOGDNA_LOOKBACK`. The set of valid values for this option are:
+The lookback strategy determines how the agent handles existing files on startup. This strategy is determined by the `LOGDNA_LOOKBACK` variable.
+
+The valid values for this option are:
+   * When set to **`none`**:
+      * lookback is disabled, and LogDNA Agent will reading new lines as those are added to the file, ignoring the lines that were written before the time the Agent restarted.
+   * When set to **`smallfiles`** (default):
+       * If there is information in the “state file”, use the last recorded state. 
+       * If the file is not present in the “state file” and the file is less than 8KiB, start at the beginning. If the file is larger than 8KiB, start at the end. 
+   * When set to **`start`**:
+    * If there is information in the “state file”, use the last recorded state. 
+    * If the file is not present in the “state file”, start at the beginning. 
```

Comment:
I think the indentation was messing up the sublist.

```suggestion
       * If there is information in the “state file”, use the last recorded state. 
       * If the file is not present in the “state file”, start at the beginning. 
```

Reply:
Hmmm let me try again, becuse I need those to be level-two bullets, so mor eindented.

Reply:
This issue persists after the latest changes.